### PR TITLE
[TOMEE-4053] Dependency properties cleanup, step 3 on branch 8.x

### PR DESCRIPTION
**Pull request by sultan**
this one is about tck tests and microprofile api & impl.

### CODE REVIEW

**Comment by sultan** [comment]
there may be room in another step to factorize more the microprofile api & tck dependencies and exclusions

File: tck/microprofile-tck/pom.xml [hunk 1]
```diff
@@ -63,4 +63,134 @@
       </modules>
     </profile>
   </profiles>
+
+  <dependencyManagement>
```
Comment (rzo1) [inline tck/microprofile-tck/pom.xml:67]: Any reason for adding them under `dependencyManagement` in the `microprofile-tck`  again? The only difference is the `scope = test`, which is decleared on each artifact in the related sub-modules?

File: tomee/tomee-microprofile/mp-common/pom.xml [hunk 2]
```diff
@@ -27,7 +27,61 @@
   <artifactId>mp-common</artifactId>
   <name>TomEE :: TomEE :: MicroProfile Common</name>
 
+  <dependencyManagement>
```
Comment (rzo1) [inline tomee/tomee-microprofile/mp-common/pom.xml:30]: same comment as above

**Comment by rzo1** [comment]
CI Build is running for this one

**Comment by sultan** [comment]
yes i can do better on this part, maybe factorize the exclusions or remove the doubling of deps managements

**Comment by rzo1** [comment]
Any updates?

**Comment by sultan** [comment]
sorry for the lack of update, i'll have time to rework the PR this weekend or after the daily job. it should not take me much time, but i'll switch the draft to ready only when ok.

**Comment by sultan** [comment]
all the versions are locked in a parent pom when desirable. there are dependency management sections for MP in three pom:
* root pom.xml for MP API
* mp-common pom.xml for MP Geronimo Impl
* microprofile-tck for MP TCK tests

File: tck/microprofile-tck/pom.xml (hunk 1 above)
Comment (sultan) [reply tck/microprofile-tck/pom.xml:67]: i believe this is not a duplicate, there are artifacts for API and more artifacts for TCK tests.

File: tomee/tomee-microprofile/mp-common/pom.xml (hunk 2 above)
Comment (sultan) [reply tomee/tomee-microprofile/mp-common/pom.xml:30]: i believe this is not a duplicate, there are artifacts for API and more artifacts for Impl.

**Review by rzo1 — APPROVED** [review]

**Pull request merged**